packages/hardhat-core/src/internal/hardhat-network/provider/fork/ForkStateManager.ts

```diff
@@ -266,15 +266,6 @@ export class ForkStateManager implements EIP2929StateManager {
       throw notCheckpointedError("commit");
     }
     this._stateCheckpoints.pop();
-
-    const storageMap = this._accessedStorage.pop();
```

Comment:
Not necessary anymore?

Reply:
Yes. The `_accessedStorageMerge` function doesn't exist anymore in the StateManager classes.

This also made me realize that the `_accessedStorage` and `_accessedStorageReverted` properties are also not necessary anymore.